docs/index.md

```diff
@@ -933,6 +933,19 @@ representing the current date. This date
 will not chanage during the execution of
 a single recipe.
 
+#### `now`
+
+The `now` variable returns a datetime
+representing the current moment. Experimentally,
```

Comment:
Would it make sense to specifically mention that this returns the value of `datetime.now()`? I don't don't if the target audience is savvy enough to know what that means.

Reply:
I think not, in general. I'll see what Peter says.